src/Elcodi/Plugin/StoreTemplateBundle/Resources/views/Pages/product-view-item.html.twig

```diff
@@ -90,5 +90,7 @@
 {% endblock content %}
 
 {% block prefooter %}
-    {% include "@StoreTemplate/Modules/_product-related.html.twig" with {'products' : related_products} %}
+    {{ render(url('store_product_related', {
+        'id': product.id
+    })) }}
```

Comment:
I usually don't use the `url` function at all to prevent requests problems when switching from `render` to `render_hinclude`.
Anyway probably this isn't a related case.